CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 0.65.0 / 2022-05-04
+
+The main change introduced by this release is the new v1alpha1 `ScrapeConfig` CRD.
+This implements the [proposal](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/proposals/202212-scrape-config.md)
+documented in [#5279](https://github.com/prometheus-operator/prometheus-operator/pull/5279)
+and provides a Kubernetes native API to create and manage additional scrape configurations.
+
+* [FEATURE] Add the `status` subresource for the `ThanosRuler` CRD. #5520
+* [FEATURE] Add `spec.web.timeout` and `spec.web.getConcurrency` to the `Alertmanager` CRD. #5478
+* [FEATURE] Add `spec.groups[].limit` to the `Prometheus` CRD. #4999
+* [FEATURE] Add ScrapeConfig CRD. #5335
+* [ENHANCEMENT] Set a default for `seccompProfile` on the operator and webhook Deployments to `RuntimeDefault`. #5477
+* [ENHANCEMENT] Add optional liveness and readiness probes to prometheus-config-reloader. This can be enabled via the `--enable-config-reloader-probes` CLI flag. #5449
+* [ENHANCEMENT] Add HTTP `scheme` validation in `Probe.spec` and `PodMonitor.spec.podMetricsEndpoints`. #5531
```

Comment:
I think that we can omit this PR because in the end, it updates only the description. We gave up enforcing the scheme value as it would have been a breaking change.